ComfortZoneApp/app/src/main/res/layout/activity_initial_comfort_actvitiy.xml

```diff
@@ -14,76 +14,93 @@
         android:layout_height="wrap_content"
         android:layout_marginHorizontal="16dp"
         android:layout_marginTop="40dp"
-        android:gravity="center"
-        android:text="Hello there! Welcome to ComfortZone, an app focused on tracking your daily comfort zones. "
-        android:textSize="20sp" />
+        android:textStyle="bold"
+        android:text="Welcome!"
+        android:textColor="@color/black"
+        android:textSize="35sp" />
```

Comment:
ideally, you also want to put dimensions/sizes/paddings/margins/any numbers you add to dimens.xml but it's less common afaik, but let's move it there as well, see how: https://stackoverflow.com/questions/29128178/how-to-use-dimens-xml-in-android